langlib/langlib-test/src/test/resources/test-src/runtimelib_test.bal

```diff
@@ -15,17 +15,52 @@
 // under the License.
 
 import ballerina/lang.runtime as runtime;
+import ballerina/jballerina.java;
 
 function getCallStackTest() returns runtime:StackFrame[] {
     return runtime:getStackTrace();
 }
 
-function getCallStacktoStringTest() returns string[] {
+function getCallStacktoStringTest() {
     runtime:StackFrame[] stackFrames = runtime:getStackTrace();
-    string[] output = [];
-    foreach runtime:StackFrame stackFrame in stackFrames {
-        output.push(stackFrame.toString());
+    assertEquality(stackFrames.length(), 3);
+    assertEquality("callableName: externGetStackTrace moduleName: ballerina.lang.runtime.0_0_1 fileName: runtime.bal lineNumber: 95", stackFrames[0].toString());
```

Comment:
These lines seem to be quite long. We usually try to maintain the same length as for Java files.

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/30998/commits/3876b1f25a63e9b504b5994027a1d6ef5f992859